### ISSUE TITLE
Please consider adding delimiter around "meta" field in logs for easier parsing

### ISSUE THREAD

**Issue by acidus99** [michael-lazar/jetforce#70]
[The code for Jetforce](https://github.com/michael-lazar/jetforce/blob/3d255a7e0b2886d487faf57bc6e771c447233487/jetforce/protocol.py#L292) currently log requests/responses using the following format, using a single whitespace character as a delimiter:
```
ip-address [timedate-stamp] "fully-qualified-URL" status-code meta response-size
```

A typical log entry looks like this:
```
1.1.1.1 [13/Dec/2022:21:15:23 +0000] "gemini://gemi.dev/cgi-bin/wp.cgi/search?foo+bar" 20 text/gemini;lang=en 3804
```

The timestamp field is surrounded by `[` and `]` delimiters, which helps with parsing, since that field can contain the whitespace character used to delimit between fields. The URL field is also surrounded by quotes, which serves the same purpose (though I've never seen Jetforce output a URL field with a literal space, its always URL encoded as `+` or `%20`, so the utility there is less).

Unfortunately, the "meta" field can contain spaces, depending on the status code, and it isn't surround with a delimiter:

* 1X - Meta will contain the user input prompt, which usually has spaces
* 4x - Human formatted error code, which usually has spaces
* 5x - Human formatted error code, which usually has spaces
* 6x - Information about the client certificate, which usually has spaces

This makes it harder to parse log lines since it's not clear where the meta line ends and the response size begins. Consider this log line from my Jetforce logs:

```
98.13.141.1 [14/Dec/2022:04:18:53 +0000] "gemini://gemi.dev/cgi-bin/wp.cgi/view" 10 Article Name? (doesn't need to be exact) 45
```

Please consider surrounding the meta field with a delimiter. I would suggest double quotes, and any double quotes in the meta field would be escaped C-style (e.g. `\"`)

**Comment by michael-lazar** [comment]
I think this is a fair point, and using double quotes like you suggest sounds like the best path forward.

The combined log format also does a similar thing with the `user-agent` and `referrer` HTTP headers, and that's probably the closest analog to what the gemini meta represents.

http://fileformats.archiveteam.org/wiki/Combined_Log_Format

**Comment by michael-lazar** [comment]
Fixed in [v0.10.0](https://github.com/michael-lazar/jetforce/releases/tag/v0.10.0)!
